### ISSUE TITLE
Request for Monster Placement in Descent

### ISSUE THREAD

**Issue by hjwilson** [NPBruce/valkyrie#1230]
### Valkyrie Version 2.4.06

I would like to ask for more options with the Mplace on Descent. At present it is only possible to use this when spawning a creature. It shows the creature momentarily and then the creature disappears. I would like to ask for a feature to be added where you can choose which monster specifically is placed on the board, and have this as something that can also be added and removed as a component to events and such.

I assume this would not be hard to do, as it is already coded in for items, search tokens etc.

Thank you so much for considering this.

**Comment by hjwilson** [comment]
It would also be fantastic to have the option to click on this and interact with it, like you can with tokens.

**Comment by NPBruce** [comment]
I don't really understand. Monster cannot be left on the screen because once they are placed Valkyrie no longer knows where they are.

Monster images could be used for tokens, but they would have to not exist as standard monsters (no activations) or this would break many things. Is this what you want?

**Comment by hjwilson** [comment]
Yes, this is what I would like. They have this happening in one of the delve levels, where a shadow dragon has "fallen over" and you can click on it to either push it or attack it. Once attacked it spawns for real. I want to do something similar to that in my level, only this time the heroes can talk to the shadow dragon or attack it. Currently I have it working where the shadow dragon appears with the spawn, then I remove the spawn and place tokens in the place where the shadow dragon should be. That doesn't look very good, however, and I was looking for a way to do this with the actual shadow dragon image.
![_private_var_mobile_Containers_Data_Application_C83E87C5-B3C3-44A4-84E4-376A816B45D8_tmp_B17A97D0-DF85-4FC4-AED7-2445A997155F_Image](https://user-images.githubusercontent.com/58572455/70383689-33a0f300-199c-11ea-8ed9-c983729313c3.jpeg)

**Comment by DelphiDie** [comment]
There is also the quest "The Troll's Treasure", where an Ettin is placed on the app screen, and has to be interacted with by clicking it. So it's definitely something that is already present in RTL. But even if it wouldn't be, it would be a good idea to add this feature.

![Screenshot](https://user-images.githubusercontent.com/26790002/70450521-2a9b4900-1aa4-11ea-9202-cdfa9cc7f37a.jpg)
